app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -219,7 +219,7 @@ Mautic.formatCode = function() {
  */
 Mautic.openMediaManager = function() {
     Mautic.openServerBrowser(
-        mauticBasePath + (typeof mauticEnv !== 'undefined' &&  mauticEnv === 'dev' ? '/index_dev.php' : '') + '/elfinder',
+        mauticBasePath + (mauticEnv === 'dev' ? '/index_dev.php' : '') + '/elfinder',
```

Comment:
This return error with image insert by legacy builder

![image](https://user-images.githubusercontent.com/462477/141692783-89eb1d73-57fb-4333-b86b-8eb302e04a3c.png)

Reply:
Good catch @kuzmany! Fixed in https://github.com/mautic/mautic/pull/8794/commits/0206837cd4bac7f1ded577da369d24ac71978b09